python/src/SquaredNormal_doc.i.in

```diff
@@ -0,0 +1,29 @@
+%feature("docstring") OT::SquaredNormal
+"Squared Normal distribution.
+
+Available constructors:
+    SquaredNormal(*mu=0.0, sigma=1.0*)
+
+Parameters
+----------
+mu : float :math:`\mu`, optional
+    Mean.
+
+    Default is :math:`\mu = 0.0`.
+
+sigma : float :math:`\sigma`, optional
+    Standard deviation.
+
+    Default is :math:`\sigma = 1.0`.
+
```

Comment:
A "Notes" section must be added, which explains the origin of this distribution and the equation of the PDF. Without this, the distribution is useless.

Reply:
;-) Not really: it allows to speed-up many formal manipulations of distributions, eg the sqr() method or the RandomMixture class. But additional documentation is often a good thing!

Reply:
Sure, I will add notes. If somebody has the corresponding latex it will be even easier. @adutfoy,  @regislebrun?

Reply:
done